packages/spright-components/src/chat/message/tests/chat-message.spec.ts

```diff
@@ -0,0 +1,40 @@
+import { html } from '@microsoft/fast-element';
+import { ChatMessage, chatMessageTag } from '..';
+import { fixture, type Fixture } from '../../../utilities/tests/fixture';
+import { ChatMessageType } from '../../types';
+
+async function setup(): Promise<Fixture<ChatMessage>> {
+    return await fixture<ChatMessage>(
+        html`<${chatMessageTag}>Some message</${chatMessageTag}>`
+    );
+}
+
+describe('ChatMessage', () => {
+    let element: ChatMessage;
+    let connect: () => Promise<void>;
+    let disconnect: () => Promise<void>;
+
+    beforeEach(async () => {
+        ({ element, connect, disconnect } = await setup());
+    });
+
+    afterEach(async () => {
+        await disconnect();
+    });
+
+    it('can construct an element instance', () => {
+        expect(document.createElement(chatMessageTag)).toBeInstanceOf(ChatMessage);
+    });
+
+    it('should have a slot element in the shadow DOM', async () => {
```

Comment:
Since we're adding slotted content in the `setup()` method (the text "Some message") we could add an assert that this content ends up in the shadow DOM too. It's not something that's likely to break but it is good to express the expected slotting behavior in a test.